cli/kaos_cli/utils/validators.py

```diff
@@ -8,7 +8,77 @@
 import textdistance
 from kaos_cli.exceptions.exceptions import MissingArgumentError
 
-from ..constants import KAOS_STATE_DIR, DOCKER, MINIKUBE, KAOS_TF_PATH
+from ..constants import KAOS_STATE_DIR, DOCKER, MINIKUBE, KAOS_TF_PATH, TF_STATE
+
+
+class EnvironmentState:
+    """
+    This class is used to validate existence of already built environmnets
+    """
+    def __init__(self):
+        self.cloud = None
+        self.env = None
+        self.build_dir = None
+        self.tf_state_path = None
+        self.if_build_dir_exists = None
+        self.if_tfstate_exists = None
+
+    @classmethod
+    def validate_build_states(cls, cloud, env) -> "EnvironmentState":
+        env_state = cls()
+        env_state.cloud = cloud
+        env_state.env = env
+        env_dir = f"{env_state.cloud}/{env_state.env}" if env_state.cloud not in [DOCKER, MINIKUBE] \
+            else f"{env_state.cloud}"
+        build_dir = os.path.join(KAOS_TF_PATH, env_dir)
+        tf_state_path = os.path.join(KAOS_TF_PATH, env_dir, f"{TF_STATE}")
+        env_state.if_build_dir_exists = os.path.exists(build_dir)
+        env_state.if_tfstate_exists = os.path.exists(tf_state_path)
+        env_state.build_dir = build_dir
+        env_state.tf_state_path = tf_state_path
+        return env_state
+
+    def display_warning(self) -> "EnvironmentState":
```

Comment:
I would rename this function to a more expressive name, I think this is a validator and if the internal states are not valid, it will issue a `sys.exit`, maybe: `validate_tfstate_exits` or `ensure_tfstate_exists`?

Reply:
Yes, this method behaves as a validator. Have renamed this as `validate_if_tfstate_exits()`, in accordance to the class variable `if_tfstate_exits`.